src/functions/uistuff.py

```diff
@@ -0,0 +1,135 @@
+from tkinter import messagebox
+from math import trunc
+from os import listdir
+
+
+def show_point(self, label):
+    """
+    Pont frissítése
+    """
+    label.config(text=f"Cookies: {trunc(self.POINTS)}")
+
+
+def show_frame(self, cont, *args):
+    """
+    Adott ablak mutatása
+    """
+    frame = self.frames[cont]
+    frame.tkraise()
+
+    saveName = ""
+    for name in args:
+        saveName = name
+
+    # Ha a játékot mentéssel indították el
+    if saveName != "":
+        from .gamestuff import loaded_game
+        loaded_game(self, saveName)
+
+    match str(cont):
+        case "<class '__main__.MainMenu'>":
+            self.isMainMenuActive = True
+            self.isGameActive = False
+            self.isSaveActive = False
+            self.isDiagramActive = False
+            if len(listdir("saves")) == 0:
+                cont.buttonLoadGame["state"] = "disable"
+            else:
+                cont.buttonLoadGame["state"] = "active"
+        case "<class '__main__.Game'>":
+            self.isMainMenuActive = False
+            self.isGameActive = True
+            self.isSaveActive = False
+            self.isDiagramActive = False
+        case "<class '__main__.Save'>":
+            self.isMainMenuActive = False
+            self.isGameActive = False
+            self.isSaveActive = True
+            self.isDiagramActive = False
+        case "<class '__main__.Diagram'>":
+            self.isMainMenuActive = False
+            self.isGameActive = False
+            self.isSaveActive = False
+            self.isDiagramActive = True
```

Comment:
Sok a kódismétlés, lehet rövidebben is:
```suggestion
    self.isMainMenuActive = str(cont) == "<class '__main__.MainMenu'>"
    self.isGameActive = str(cont) == "<class '__main__.Game'>"
    self.isSaveActive = str(cont) == "<class '__main__.Save'>"
    self.isDiagramActive = str(cont) == "<class '__main__.Diagram'>"
    cont.buttonLoadGame["state"] = "active" if listdir("saves") else "disable"
```